docs/DEPLOY.md

```diff
@@ -41,7 +55,7 @@ then user try to deploy
 `sls deploy -s prod -r westus`
 
 1. Do we use the environment variables / creds associated with `dev` or `prod`?
-1. Someone new to the codecase see both the `serverless.yaml` and the command in a CD script, how can they tell which resourceGroup goes where.
+2. Someone new to the codecase see both the `serverless.yaml` and the command in a CD script, how can they tell which resourceGroup goes where.
```

Comment:
you should use follow markdown syntax and use `1.`.  It will auto-number on render

Reply:
Do you have some docs that specify that just `1.` is preferred? In the [markdown style guide on ordered lists](https://www.markdownguide.org/basic-syntax/#ordered-lists), it includes that as an _option_, but not the first one (another option being to just list any number at random, which will render to the same thing). Also, VSCode does an auto-complete to increment the number in markdown, which makes me think the explicit number is preferred (or at least commonly used). At the end of the day, they all render to the same thing, I was just curious if you had another source that stated that `1.` is preferred. Thanks for the feedback 😃

Reply:
You could use fibonacci sequence - it doesn't matter... will still come out as 1,2,3,4
1. a
3. b
5. c
8. d
13. e